postgres/init.sls

```diff
@@ -8,6 +8,11 @@ include:
 {{ postgres.conf_dir }}:
   file.directory:
     - makedirs: True
+    - user: postgres
+    - group: postgres
+    - dir_mode: 700
+    - clean: True
```

Comment:
@rominf Do you have some documentation about testing formulas with `unittest`?

This module looks like it now works with [test-kitchen](http://kitchen.ci/). You may want to look at that for testing.